cmd/eksctl-anywhere/cmd/installpackages.go

```diff
@@ -0,0 +1,104 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/pkg/config"
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+type installPackageOptions struct {
+	source      curatedpackages.BundleSource
+	kubeVersion string
+	name        string
+	registry    string
+}
+
+var ipo = &installPackageOptions{}
+
+func init() {
+	installCmd.AddCommand(installPackageCommand)
+	installPackageCommand.Flags().Var(&ipo.source, "source", "Location to find curated packages: (cluster, registry)")
+	if err := installPackageCommand.MarkFlagRequired("source"); err != nil {
+		log.Fatalf("Error marking flag as required: %v", err)
+	}
+	installPackageCommand.Flags().StringVar(&ipo.kubeVersion, "kubeversion", "", "Kubernetes Version of the cluster to be used. Format <major>.<minor>")
+	installPackageCommand.Flags().StringVar(&ipo.name, "name", "", "Custom name of the curated package to install")
```

Comment:
Why can't this be `package` instead? name makes me think of namespace, and I think it is okay to be like `eksctl anywhere install packages -p my-curated-package`. This is similar to how kubectl does `kubectl create deployment my-dep --image=busybox`

Reply:
A package has a conceptual meaning in curated packages. If we were to use a different name than just name it would be packageName and -p as the short form. Would that work better than name? Thoughts?

Reply:
I like that